pkg/platform/packet/packet.go

```diff
@@ -294,3 +330,117 @@ func (c *config) checkWorkerPoolNamesUnique() hcl.Diagnostics {
 
 	return diagnostics
 }
+
+func (c *config) checkReservationIDs() hcl.Diagnostics {
```

Comment:
```suggestion
func (c *config) validateReservationIDs() hcl.Diagnostics {
```

Reply:
Big no. All other functions are called `check*`, like `checkNotEmptyWorkers`, `checkWorkerPoolNamesUnique`. It's just not coherent with existing code

Reply:
Please re-open of you want to discuss this further